src/App.tsx

```diff
@@ -138,15 +138,31 @@ const SettingsComponent: React.FC = () => {
           </fieldset>
           <fieldset>
             <label>
-              Model:
+              Select Model:
               <br />
               <input
                 type="text"
-                value={defaultOpenAiModel}
                 onChange={(e) => setDefaultOpenAiModel(e.target.value)}
+                list="modelOptions"
               />
             </label>
+            <datalist id="modelOptions">
+              <option value="gpt-4o" />
+              <option value="gpt-4o-mini" />
+              <option value="gpt-4o-turbo" />
+              <option value="gpt-4" />
+              <option value="gpt-3.5-turbo" />            
+            </datalist>
             <button onClick={handleModelReset}>Reset to default</button>
+            <p>Here are some available ChatGPT model options:</p>
+            <p>
+            <b>gpt-4o </b>, 
+            <b>gpt-4o-mini </b>, 
+            <b>gpt-4-turbo </b>, 
+            <b>gpt-4 </b>, and <b>gpt-3.5-turbo</b>
+            </p>
+          
+           <p>For a complete list of models, please check the <a href="https://platform.openai.com/docs/models" target="_blank">ChatGPT models page</a>.</p>
```

Comment:
thank you for adding `target="_blank"` attribute, can you also add `rel="noopener noreferrer"` for enhanced security?

Context:

The `rel=noopener` attribute instructs the browser to navigate to the target resource without granting the new browsing context access to the document that opened it.

The `rel=noreelements` instructs the browser, when navigating to the target resource, to omit the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) header and otherwise leak no referrer information. This way the user privacy is not leaked.

Including both ensures security and privacy when navigating to external pages.

Reply:
Added rel="noopener noreferrer" now in a2bc78ef879d93c805c60e5d1cb33b901152d85c 
made a note now about security implications of not having this here!